write-fonts/src/pens.rs

```diff
@@ -221,16 +230,14 @@ fn flush_subpath<T: Pen>(commands: &[PenCommand], pen: &mut T) -> Result<(), Con
                 x: end_x,
                 y: end_y,
             },
-            // Close is a line from (end_x,end_y)=>(start_x, start_y) so reversed it's a line to end x/y
-            PenCommand::Close => PenCommand::LineTo { x: end_x, y: end_y },
+            PenCommand::Close => {
+                panic!("Subpath should have 0 or 1 close, and it should already have been removed")
```

Comment:
I would prefer this be an error, akin to subpath does not start with moveto above

Reply:
Since you are headed on vacation I will adjust then merge.

Reply:
Done